src/plugins/prebuilt-library/ai-wizard/constants/index.js

```diff
@@ -161,3 +136,49 @@ export const ENTITY_TO_NAME = {
 	'ORGANIZATION': __( 'Organization Name', 'kadence-blocks' ),
 };
 
+export const LOCATION_BUSINESS_ADDRESS = 'Business Address';
+export const LOCATION_SERVICE_AREA = 'Service Area';
+export const LOCATION_ONLINE_ONLY = 'Online Only';
+
+export const LOCATION_TYPES = [
+	{
+		icon: store,
+		text: __( 'Business Address', 'kadence-blocks' ),
+		value: LOCATION_BUSINESS_ADDRESS,
+		help: __( 'E.g.: 1234 Street #1, Chicago, IL 60076, USA', 'kadence-blocks' ),
+		placeholder: __( 'Street Adress, City, State, Zipcode, Country', 'kadence-blocks' ),
+	},
+	{
+		icon: mapMarker,
+		text: __( 'Service Area', 'kadence-blocks' ),
+		value: LOCATION_SERVICE_AREA,
+		help: __( 'E.g.: Chicago, USA', 'kadence-blocks' ),
+		placeholder: __( 'District, City, State, Zipcode, Country', 'kadence-blocks' ),
+	},
+	{
+		icon: desktop,
+		text: __( 'Online Only', 'kadence-blocks' ),
+		value: LOCATION_ONLINE_ONLY,
+		help: '',
+		placeholder: '',
+	},
+];
+
+export const MISSION_STATEMENT_GOAL = 300;
+
+export const MISSION_STATEMENT_STATUS = {
+	'weak': {
+		color: '#DF3416',
+		message: __( 'For better, more accurate copy enter a bit more content.', 'kadence-blocks' ),
+	},
+	'medium': {
+		color: '#B35F00',
+		message: __( 'This is a great start. Try adding to your mission statement a bit more if you can.', 'kadence-blocks' ),
```

Comment:
Can we update this to "This is a great start. Try adding a bit more if you can." That way we don't confuse by saying "mission statement"

Reply:
I've updated this content.